.gofmt.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Would you mind adding go vet as well, like we have here? https://github.com/GoogleCloudPlatform/debian-docker/blob/master/check-fmt.sh#L8